drivers/bridge/bridge.go

```diff
@@ -145,6 +146,159 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// FromMap retrieve the configuration data from the map form.
+func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
```

Comment:
+1 I personally like to avoid this `FromMap` if possible. 
Yes, Docker changes must assume the structure of `NetworkConfiguration` and populate the data-types that map the structure. But, as I see it, Bridge driver is a special case due to backward compatibility.
So, when it comes to trade-offs, I would prefer to take the approach of using `GenerateFromModel` over having to change the existing code in Docker.